pandas/core/arrays/interval.py

```diff
@@ -397,10 +348,16 @@ def from_breaks(cls, breaks, closed="right", copy=False, dtype=None):
     def from_arrays(cls, left, right, closed="right", copy=False, dtype=None):
         left = maybe_convert_platform_interval(left)
         right = maybe_convert_platform_interval(right)
+        if len(left) != len(right):
+            raise ValueError("left and right must have the same length")
 
-        return cls._simple_new(
-            left, right, closed, copy=copy, dtype=dtype, verify_integrity=True
-        )
+        closed = closed or "right"
+        left, right = _maybe_cast_inputs(left, right, copy, dtype)
+        combined = _get_combined_data(left, right)
+
+        result = cls._simple_new(combined, closed)
+        result._validate()
```

Comment:
is there a reason we don't call _validate in _simple_new (with a passed in verify_integrity arg)?

Reply:
bc then it wouldnt be simple